code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -412,7 +412,7 @@
 		/obj/item/crowbar/cyborg/power, // Skyrat ADDITION - Combines Crowbar and Wirecutters into one
 		/obj/item/multitool/cyborg, //Skyrat ADDITION - Adds multitool for easier access
 		/obj/item/borg/cyborg_omnitool/engineering,
-		///obj/item/borg/cyborg_omnitool/engineering, //Skyrat REMOVAL
+		/obj/item/borg/cyborg_omnitool/engineering,
```

Comment:
Here's an example of the merge being weird

Reply:
Doubt I would make it leave 2 omnitools intentionally so probably some merge fuckery